mavedb_mapping/transcript_selection_helper.py

```diff
@@ -8,8 +8,8 @@
 from Bio.Seq import Seq
 from ga4gh.vrs import models
 
-sr = SeqRepo("/usr/local/share/seqrepo/latest", writeable=True)
 environ["UTA_DB_URL"] = "postgresql://uta_admin:uta@localhost:5432/uta/uta_20210129"
+sr = SeqRepo("/usr/local/share/seqrepo/latest", writeable=True)
```

Comment:
I noticed that this pattern shows up in a few places, where you use seqrepo and/or UTA. Universally, you will want to get rid of hard-coding paths. But you might also consider adding these interfaces in a single location for import, e.g. from `__init__.py`.